tst/EnergyPlus/unit/CMakeLists.txt

```diff
@@ -252,14 +252,14 @@ set(test_src
     ZoneTempPredictorCorrector.unit.cc
     main.cc)
 
-set(test_dependencies energyplusapi)
+set(test_dependencies energypluslib)
 
 if(LINK_WITH_PYTHON)
   add_definitions("-DLINK_WITH_PYTHON")
   find_package(PythonLibs 3 REQUIRED)
   include_directories(${Python_INCLUDE_DIRS})
   if(CMAKE_HOST_UNIX)
-    list(APPEND test_src api/datatransfer.unit.cc)
+    #list(APPEND test_src api/datatransfer.unit.cc) # Requires API; will be moved
```

Comment:
Definitely, the tests here just need to be pulled into external API calling scripts.  We have a collection of those, thanks to @jmarrec, so the stuff in here may not even be necessary anymore.